test/integration/scheduler_perf/dra/templates/resourceslice-partitionable.yaml

```diff
@@ -1,5 +1,5 @@
 kind: ResourceSlice
-apiVersion: resource.k8s.io/v1beta1
```

Comment:
I'm surprised this was part of this PR... removing the v1alpha3 type shouldn't require switching this to v1beta2, right?

Reply:
if it's a cleanup, it's ~fine if it's included in this PR, just wanting to make sure this wasn't required as a surprising side effect of dropping v1alpha3 or something

Reply:
/hold
to double check this, can unhold if this was just a cleanup and not *required*

Reply:
The scheduler_perf tests were still using the v1alpha3 types and API through a generic client. Should have been updated before, but that was overlooked and noted only now when searching for references to v1alpha3 - so a cleanup.